app/navigation/IndividualProtectorNavigator.scala

```diff
@@ -57,14 +57,14 @@ class IndividualProtectorNavigator @Inject()() extends Navigator {
         case IdCardDetailsYesNoPage => ua =>
           yesNoNav(ua, IdCardDetailsYesNoPage, rts.IdCardDetailsController.onPageLoad(mode), rts.StartDateController.onPageLoad())
       }
-//      case CheckMode => {
-//        case NationalInsuranceNumberPage | UkAddressPage | NonUkAddressPage | PassportDetailsPage | IdCardDetailsPage => ua =>
-//          checkDetailsRoute(ua)
-//        case AddressYesNoPage => ua =>
-//          yesNoNav(ua, AddressYesNoPage, rts.LiveInTheUkYesNoController.onPageLoad(mode), checkDetailsRoute(ua))
-//        case IdCardDetailsYesNoPage => ua =>
-//          yesNoNav(ua, IdCardDetailsYesNoPage, rts.IdCardDetailsController.onPageLoad(mode), checkDetailsRoute(ua))
-//      }
+      case CheckMode => {
+        case NationalInsuranceNumberPage | UkAddressPage | NonUkAddressPage | PassportDetailsPage | IdCardDetailsPage => ua =>
+          checkDetailsRoute(ua)
+        case AddressYesNoPage => ua =>
+          yesNoNav(ua, AddressYesNoPage, rts.LiveInTheUkYesNoController.onPageLoad(mode), checkDetailsRoute(ua))
+        case IdCardDetailsYesNoPage => ua =>
+          yesNoNav(ua, IdCardDetailsYesNoPage, rts.IdCardDetailsController.onPageLoad(mode), checkDetailsRoute(ua))
+      }
```

Comment:
UkAddressPage and NonUkAddressPage are part of simple navigation so don't need to be in here. This applies to line 53 as well